ledger/src/blockstore/column.rs

```diff
@@ -735,6 +735,30 @@ impl ColumnName for columns::SlotMeta {
 }
 impl TypedColumn for columns::SlotMeta {
     type Type = blockstore_meta::SlotMeta;
+
+    fn deserialize(data: &[u8]) -> Result<Self::Type> {
```

Comment:
Unfortunately, we have at least one caller that deserializes directly here:
https://github.com/anza-xyz/agave/blob/1ceeab0548d5db0bbcf7dab7726c2ffb4180fa76/ledger/src/blockstore.rs#L655-L665

Could you also please double check if there are any other spots that might need update. I think other potential offenders should only be in `blockstore.rs` (look for `.meta_cf`), but a quick pass on the following two spots would be good too:
- ledger-tool
- Repair code

Reply:
Some other general cleanup that'd be nice:
- Update any `deserialize()` calls in `blockstore.rs` to use invoke the column's `deserialize()` function
 - `Blockstore::get_recent_perf_samples()` has similar fallback logic that looks like it would be good to shift into the column specific logic in `column.rs`

Neither of these items are related to / a blocker for this PR, but mentioning them while we are looking/thinking about this and incase you'd be willing to pick them up 😄